examples/contrib/oed/results_parser_infograd.py

```diff
@@ -16,30 +23,23 @@ def main(name, sampling_interval):
     xi_history = results['xi_history']
     est_eig_history = results['est_eig_history']
     eig_history = results.get('eig_history')
-    eig_heatmap = results.get('eig_heatmap')
-    heatmap_extent = results.get('extent')
+    eig_lower = results.get('lower_history')
+    eig_upper = results.get('upper_history')
 
-    if eig_heatmap is not None:
-        plt.imshow(eig_heatmap, cmap="gray", extent=heatmap_extent, origin='lower')
-    x, y = xi_history[::sampling_interval, 0].detach(), xi_history[::sampling_interval, 1].detach()
-    plt.scatter(x, y, c=torch.arange(x.shape[0]), marker='x', cmap='summer')
-    plt.show()
-    print(xi_history[-1,...])
-
-    plt.plot(est_eig_history.detach().clamp(min=0).numpy())
-    if eig_history is not None:
-        plt.plot(eig_history.detach().numpy())
-        print("Final true EIG", eig_history[-1].item())
-        print("Max EIG over surface", eig_heatmap.max().item())
-        print("Discrepancy", (eig_heatmap.max() - eig_history[-1]).item())
-    plt.legend(["Approximate EIG", "True EIG"])
```

Comment:
don't delete, allow different parts of the parser to work for different availabilities